include/scrimmage/plugins/network/SphereNetwork/SphereNetwork.h

```diff
@@ -53,6 +53,16 @@ class SphereNetwork : public scrimmage::Network {
                                             const scrimmage::PluginPtr &sub_plugin) override;
     double range_;
     double prob_transmit_;
+
+    // Attributes of a boundary plane that blocks communication. Example use
+    // would be putting a boundary at a water/air intersection such that
+    // communications wouldn't go between water and the surface.
+    double comms_boundary_altitude_;
```

Comment:
Can you add appropriate default values for the variables in the header file? (e.g., bool filter_comms_plane_ = false;)

Reply:
@SyllogismRXS Do you want me to add default values in the header for the doubles? i.e. for comms_boundary_altitiude_ and comms_boundary_epsilon_. If they aren't in the params file I have them default to quietNaN right now when they are being read in.